package.json

```diff
@@ -54,7 +54,7 @@
     "@types/lodash": "^4.14.161",
     "@types/minimist": "^1.2.0",
     "@types/nedb": "^1.8.11",
-    "@types/node": "^13.0.0",
+    "@types/node": "^13.13.4",
```

Comment:
could you please explain the reason behind this change?

Reply:
The version 13.0.0 not exists in the NPM package versions https://www.npmjs.com/package/@types/node.